### ISSUE TITLE
Listing B.2 not up-to-date

### ISSUE THREAD

**Issue by h4sh3d** [jacquesd/master-thesis#5]
The listing B.2 use the old values for `r` and `s`.

**Comment by 0xjac** [comment]
Good catch, however this was the "current" value at the time of the deadline. Lots as changed since and the whole section 6 is now outdated. Time permitting I will try to provide a second version of the thesis with updates.
